### DISCUSSION TITLE
On the response time to support tickets.

### DISCUSSION THREAD

**Discussion by tiagox** [community/community#127237] [GitHub Education] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, I'm concern about the response time to the supports tickets.

The specific case is that me and some colleges are using GitHub Classroom for our university class. We've created an organization that, a couple of days ago, was flagged without any notice or hint of what we might be infringing from the CoC, the Community Guidelines or any other policy that might caused the flagging.

My concerns started when I searched on the community forum and I only saw cases of people and organization being flagged for +30 days (at least). I've followed the steps and I was told... created the request ticket as asked and wait for a response that's not happening.

This flagging has paralyzed the development of our class, students stopped being able to accept new assignments. Classroom grades are nearly inaccessible.

I haven't seen any real help on this forum for this kind of comments, but I hope this can help improve the platform, we're planning to move away from GitHub for the next semester. And, just for you to know. We were doing free publicity for GitHub to our students for so long, but unluckily for you that won't be the case anymore.

The most important aspect that really concerning it's **the absolute absence of communication from the GitHub Support Team**. If there's any.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @tiagox 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by tiagox** [comment]
Just for the record it's been 22 days without an answer from the support team.
